### PR TITLE
Bach: Fix sorting for Series.describe

### PR DESCRIPTION
**Pull request by KathiaBarahona**
`Series.describe` was ignoring the sorting set in `DescribeOperation`. Created `SeriesDescribeOperation` that handles the sorting of the final result.

### CODE REVIEW

File: bach/tests/functional/bach/test_series_describe.py [hunk 1]
```diff
@@ -17,22 +17,26 @@ def test_categorical_describe() -> None:
...
+    result = series.describe(percentiles=[0.88, 0.5, 0.75])
     assert isinstance(result, Series)
+    assert len(result.sorting_keys) == 1
 
     expected = pd.Series(
-        index=pd.Index(['count', 'mean', 'std', 'min', 'max', 'nunique', 'mode', '0.5'], name='__stat'),
-        data=[3., 2., 1., 1., 3., 3., 1., 2.],
+        index=pd.Index(
+            ['count', 'mean', 'std', 'min', 'max', 'nunique', 'mode', '0.5', '0.75', '0.88'],
+            name='__stat'
```
[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_series_describe.py:39]: Here an issue for that https://github.com/objectiv/objectiv-analytics/issues/999

File: bach/tests/functional/bach/test_series_describe.py [hunk 2]
```diff
@@ -17,22 +17,26 @@ def test_categorical_describe() -> None:
         ],
         expected_data=[
```
[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_series_describe.py:20]: Here the fix https://github.com/objectiv/objectiv-analytics/pull/996/commits/c0133b0fcfa7ee9ae894afba5e14a413c8e33f5e

**Review by vard-obj — APPROVED** [review]
lgtm

File: bach/bach/operations/describe.py [hunk 3]
```diff
@@ -237,3 +243,27 @@ def _calculate_percentiles(self) -> Optional[DataFrame]:
...
+
+
+class DataFrameDescribeOperation(DescribeOperation[DataFrame]):
+    """
+    To use: first instantiate this class,  then call the instantiated object as a function,
+    e.g. `df_described = DataFrameDescribeOperation(obj=df)()`
+    """
+    def _get_final_described_result(self, describe_df: DataFrame) -> DataFrame:
+        describe_df = describe_df.sort_values(by=f'{self.STAT_SERIES_NAME}_position')
+        all_described_series = [
```
[the remark this answers is not in the document]
Comment (thijs-obj) [reply bach/bach/operations/describe.py:255]: If so, better to just use that imho @KathiaBarahona

[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/bach/operations/describe.py:255]: You are right

[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/bach/operations/describe.py]: changed here https://github.com/objectiv/objectiv-analytics/pull/996/commits/38ad096e4ce422ddb8f9e2038991ea95f5d2f16b

**Review by thijs-obj — APPROVED** [review]
lgtm

**Pull request merged**